packages/tools/src/tools/segmentation/strategies/utils/initializeTracking.ts

```diff
@@ -0,0 +1,21 @@
+import type { OperationData, InitializedOperationData } from '../BrushStrategy';
+
+/**
+ * Sets up tracking for use by preview and other services.
+ * This sets up a  TRACKING value in the strategy specific configuration which
+ * stores the original value for updated pixels, and allows the changes to be
+ * applied (eg for a preview), reverted, or acted on in other ways.
+ */
+export default {
+  createInitialized: (enabled, operationData: InitializedOperationData) => {
```

Comment:
is `enabled` `enabledElement`? please rename it to enabledElement, I was struggling with what is this

Reply:
Can we use a short name for things that aren't used?  Like na for not applicable.

Reply:
please no!
I had to dig into the code to understand oh this is enabledElement.